pkgs/applications/radio/rtl-wmbus/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, stdenv, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  version = "unstable-2022-05-07";
+  pname = "rtl-wmbus";
+
+  src = fetchFromGitHub {
+    owner = "xaelsouth";
+    repo = pname;
+    rev = "388e7adb8e87caba2fcd50de979a6473e66d475b";
+    sha256 = "sha256-PuKoF57wUKJDZu9EwpdRP34+MBPnhEp4VKOfEE/Kog0";
+  };
+
+  makeFlags = [
+    "COMMIT_HASH="
+    "TAG=${src.rev}"
+    "BRANCH="
+    "CHANGES="
+  ];
+
+  preBuild = ''
+    substituteInPlace Makefile \
+      --replace "CC=gcc" "CC?=gcc" \
+      --replace "/usr" ""
+  '';
+
+  installFlags = [ "DESTDIR=$(out)" ];
+
+  meta = with lib; {
+    description = "Software defined receiver for Wireless M-Bus with RTL-SDR";
+    homepage = "https://github.com/xaelsouth/rtl-wmbus";
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ snicket2100 ];
+    platforms = platforms.unix;
```

`nix run` fails to run this because it's looking for a binary named `rtl-wmbus`, but it builds`rtl_wmbus` instead.

Tried it with:
```
nix run 'github:nixos/nixpkgs?ref=refs/pull/221814/head#rtl-wmbus'
```

You can add a [mainProgram](https://nixos.org/manual/nixpkgs/stable/#var-meta-mainProgram) attribute to fix that.

```suggestion
    mainProgram = "rtl_wmbus";
    maintainers = with maintainers; [ snicket2100 ];
    platforms = platforms.unix;
```